### PR TITLE
Move text on button press

### PR DESCRIPTION
**Pull request by fstr0**
Modify padding on active button to move the text 1px down and 1px right to replicate behaviour in Win98.

This fixes issue #34.

### CODE REVIEW

File: build/98.css [hunk 1]
```diff
@@ -102,6 +102,7 @@ button:not(:disabled):active {
   box-shadow: inset -1px -1px #ffffff,
     inset 1px 1px #0a0a0a, inset -2px -2px #dfdfdf,
     inset 2px 2px #808080;
+  padding: 2px 11px 0 13px;
```
Comment (DiskJunky) [inline build/98.css:105]: I had the exact same fix. Just to note for posterity on some of the idiosyncrasies of the fix:
* `2px` was required due to Chrome not moving a `1px` value. There appears to be a conflicting rule in css resolution there
* `0px` for the bottom as negative values aren't valid in FireFox and Chrome for `padding`. Interestingly, full `padding-top`, `right`, etc. does actually work. Issue is specifically with the condensed `padding` tag for both browsers.

**Comment by jdan** [comment]
Is there a way to do this without padding? I worry about someone adding their own padding (for larger buttons) and forgetting to add/subtract one on `:active`

**Comment by tpenguinltg** [comment]
Perhaps you can make use of an invisible border instead since `box-sizing` is set to `border-box`? As far as I can tell, the border has been removed on buttons anyway, and I don't see any legitimate use for a downstream user to add it back if they want to keep the theme.

**Comment by michaeldll** [comment]
> Perhaps you can make use of an invisible border instead since `box-sizing` is set to `border-box`? As far as I can tell, the border has been removed on buttons anyway, and I don't see any legitimate use for a downstream user to add it back if they want to keep the theme.

I had a go at implementing your suggestion on #79 :)

**Comment by jdan** [comment]
@fstr0 I'm happy to merge this as-is, since I don't think we can do better than adjusting the padding :) Wanna rebase off master?

**Comment by fstr0** [comment]
@jdan I think that is done now? I'm a GitHub noob so let me know if I did it wrong

File: build/98.css [hunk 2]
```diff
@@ -0,0 +1,527 @@
+/**
```
Comment (jdan) [inline build/98.css:1]: These have since been removed from version control :) Wanna rm this?

File: docs/98.css [hunk 3]
```diff
@@ -0,0 +1,527 @@
+/**
```
Comment (jdan) [inline docs/98.css:1]: These have since been removed from version control :) Wanna rm this?

**Comment by jdan** [comment]
@fstr0 thanks so much <3

**Pull request merged**